auth0/v3/authentication/token_verifier.py

```diff
@@ -0,0 +1,225 @@
+import json
+import time
+
+import jwt
+import requests
+
+from auth0.v3.exceptions import TokenValidationError
+
+
+class SignatureVerifier():
+    DISABLE_JWT_CHECKS = {
```

Comment:
These need to be disabled here in order to only validate the signature of the token